git_commands.go

```diff
@@ -2,39 +2,72 @@ package main
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"os/exec"
 	"strings"
 )
 
 func (c *GitCommand) PullAll(branch string) ([]byte, error) {
-	cmd := exec.Command("git", "-c", "color.status=always", "pull", "origin", fmt.Sprintf("%s", branch))
+	// in the event that the branch is prefixed with origin, we will parse to remove 'origin'
+	var cmd *exec.Cmd
+	if strings.HasPrefix(branch, "origin") {
+		branchName := RemoveOriginFromBranchName(branch)
+		cmd = exec.Command("git", "pull", "origin", fmt.Sprintf("%s", branchName))
+	} else {
+		cmd = exec.Command("git","pull", "origin", fmt.Sprintf("%s", branch))
+	}
+
 	cmd.Dir = c.Dir
-	output, err := cmd.Output()
+	output, err := cmd.CombinedOutput()
```

Comment:
CombinedOutput populated Stderr, which is helpful when rendering information for users.